docs/usage-js.rst

```diff
@@ -56,7 +56,7 @@ Let’s have a look at an example configuration first. (Please also see the :ref
 
        'require': null, // String, When using nodejs hooks, require the given module before executing hooks
 
-       'no-color': false,
+       'color': true,
```

Comment:
Unsure about the behaviour here, generally in CLI tools it is good idea to detect the default value based on if the shell is TTY (example at https://github.com/kylef/Commander/blob/314f8d7455e25e60e9e9b850040eebd0a87f2527/Sources/Commander/Error.swift#L42-L56, although in this case you should simply be able to use an `isatty` method instead of the TERM parsing, that works around Xcode specific logger that don't apply here).

That way if I do something like:

```
dredd > somefile
```

Colours will automatically be disabled and the written file won't include ANSI codes.

Of course, `--colour` and `--no-colour` would override from automatic behaviour.

Reply:
Yes, I realized this one when reading one article about CLI apps later after issuing the Pull Request already. I'd like to do it exactly as you write, I think it's [easy to detect TTY in the Node](https://nodejs.org/api/process.html#process_a_note_on_process_i_o).

I wasn't sure though whether to snowball it here or to make the change separately. Also, I'm not sure the `stdout` and `stderr` are completely well separated (they weren't even before this change, the logger seems to print each message to whatever it feels is right... I want to fix that as a next step as well), which could make this complicated now.